integration_tests/test_mempool.py

```diff
@@ -56,11 +54,12 @@ def test_mempool(cronos_mempool):
     print(f"all pending tx hash after block: {all_pending}")
     assert len(all_pending) == 0
 
-    # check transaction
-    block_num_0 = w3.eth.get_block_number()
-    print(f"block number start: {block_num_0}")
     nonce_begin = w3.eth.get_transaction_count(address_from)
 
+    # wait block update
+    block_num_0 = wait_for_new_blocks(cli, 1, sleep=0.1)
```

Comment:
could mv assign `block_num_0` right before `send_raw_transaction` 1st tnx

Reply:
sounds good, we can build the raw tx in advance.